bindings/python/iota_wallet/wallet.py

```diff
@@ -198,8 +213,10 @@ def store_mnemonic(self, mnemonic):
         """Store mnemonic.
         """
         return self._send_cmd_routine(
-            'storeMnemonic',
-            mnemonic
+            'storeMnemonic', {
+                'mnemonic': mnemonic
+            }
+
         )
```

Comment:
Commenting at a random place: do we not have EmitTestEvent in python ?

Reply:
No